### PR TITLE
New package: giara-0.3

### PR DESCRIPTION
**Pull request by paper42**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/python3-prawcore/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+short_desc="Low-level communication layer for PRAW 4+"
+maintainer="Paper <paper@tilde.institute>"
+license="BSD-2-Clause"
+homepage="https://github.com/praw-dev/prawcore"
+changelog="https://raw.githubusercontent.com/praw-dev/prawcore/master/CHANGES.rst"
+distfiles="https://github.com/praw-dev/prawcore/archive/v$version.tar.gz"
+checksum=54b32531a16f9335b02bdd7d69027712a8181428dfe43191c1a8b9510e413fca
+
+do_check() {
+	# needs python3-betamax, python3-betamax-matches python3-betamax-serializers
```
Comment (ericonr) [inline srcpkgs/python3-prawcore/template:18]: Nit:

```suggestion
	# needs python3-betamax, python3-betamax-matches, python3-betamax-serializers
```

File: srcpkgs/giara/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'giara'
+pkgname=giara
+version=0.3
+revision=1
+build_style=meson
+hostmakedepends="gobject-introspection python3-setuptools pkg-config glib-devel gettext"
+makedepends="gtk+3-devel libhandy1-devel"
+depends="python3-BeautifulSoup4 python3-cairo python3-dateutil python3-gobject
+ python3-mistune python3-Pillow python3-praw gtksourceview4 webkit2gtk"
```
Comment (ericonr) [inline srcpkgs/giara/template:9]: Hmm, another dependant on mistune... Do you know if they have plans to support mistune 2?

Comment (paper42) [reply srcpkgs/giara/template:9]: I don't know, but isn't mistune 2 still in alpha?

Comment (ericonr) [reply srcpkgs/giara/template:9]: It is, and I'm no longer planning on packaging it.

**Comment by paper42** [comment]
sorry for the delay, I updated python3-praw and python3-prawcore to the latest versions

File: srcpkgs/python3-praw/template [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+license="BSD-2-Clause"
+homepage="https://github.com/praw-dev/praw/"
+changelog="https://raw.githubusercontent.com/praw-dev/praw/master/CHANGES.rst"
+distfiles="https://github.com/praw-dev/praw/archive/v$version.tar.gz"
+checksum=20804681e7aa4d2a0789663f61f8996a04455dde718ff667fb1800e3db7c8716
+
+do_check() {
+	# needs python3-betamax and python3-betamax-matches which are not packaged
+	:
+}
```
Comment (ericonr) [inline srcpkgs/python3-praw/template:20]: `make_check=no`

File: srcpkgs/python3-prawcore/template [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+homepage="https://github.com/praw-dev/prawcore"
+changelog="https://raw.githubusercontent.com/praw-dev/prawcore/master/CHANGES.rst"
+distfiles="https://github.com/praw-dev/prawcore/archive/v$version.tar.gz"
+checksum=71f216beac824a58413c35ea61aea5bef1c1b90d6ae2a01d9640c6a9b73a0e56
+
+do_check() {
+	# needs python3-betamax, python3-betamax-matches, python3-betamax-serializers
+	# and python3-textfixtures which are not packaged
+	:
+}
```
Comment (ericonr) [inline srcpkgs/python3-prawcore/template:21]: Same here.

**Comment by paper42** [comment]
I added a new patch which fixes initial login.

File: srcpkgs/giara/template [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+short_desc="Reddit client for GNOME"
+maintainer="Paper <paper@tilde.institute>"
+license="GPL-3.0-or-later"
+homepage="https://gitlab.gnome.org/World/giara"
+distfiles="https://gitlab.gnome.org/World/giara/-/archive/$version/giara-$version.tar.gz"
+checksum=111d9f2aa46c58cf5cfe3e677988a2726701ca7ba1cc94948176d2345d80ff7f
+patch_args="-Np1"
```
Comment (ericonr) [inline srcpkgs/giara/template:16]: No longer needed.

File: srcpkgs/giara/patches/fix-login.patch [hunk 6]
```diff
@@ -0,0 +1,28 @@
+https://gitlab.gnome.org/World/giara/-/merge_requests/12
```
Comment (ericonr) [inline srcpkgs/giara/patches/fix-login.patch:1]: Could ask upstream for release?

File: srcpkgs/python3-prawcore/template [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+hostmakedepends="python3-setuptools"
+depends="python3-requests"
+short_desc="Low-level communication layer for PRAW 4+"
+maintainer="Paper <paper@tilde.institute>"
+license="BSD-2-Clause"
+homepage="https://github.com/praw-dev/prawcore"
+changelog="https://raw.githubusercontent.com/praw-dev/prawcore/master/CHANGES.rst"
+distfiles="https://github.com/praw-dev/prawcore/archive/v$version.tar.gz"
+checksum=71f216beac824a58413c35ea61aea5bef1c1b90d6ae2a01d9640c6a9b73a0e56
+make_check=no # needs python3-betamax, python3-betamax-matches, python3-betamax-serializers and python3-textfixtures which are not packaged
```
Comment (ericonr) [inline srcpkgs/python3-prawcore/template:16]: Line too long.

File: srcpkgs/giara/patches/fix-login.patch (hunk 6 above)
Comment (paper42) [reply srcpkgs/giara/patches/fix-login.patch]: I did, no response :(

**Pull request closed without merging**